docs/detections/rules-cross-cluster-search.asciidoc

```diff
@@ -0,0 +1,65 @@
+[[rules-cross-cluster-search]]
+== Cross-cluster search and detection rules
+
+{ref}/modules-cross-cluster-search.html[Cross-cluster search] is an {es} feature that allows one cluster (the _local_ cluster) to query data in a separate cluster (the _remote_ cluster). {elastic-sec}'s detection rules can perform cross-cluster search to query data in remote clusters.
+
+[discrete]
+[[set-up-ccs-rules]]
+=== Set up cross-cluster search in detection rules
+
+This section explains the general process for setting up cross-cluster search in detection rules. For specific instructions on each part of the process, refer to the linked documentation.
+
+. On the local cluster, {ref}/remote-clusters-connect.html[set up a connection] to the remote cluster. Note the unique name that you give to the remote cluster, because you'll need to use it throughout this process.
+
+. On both the local and remote clusters, {ref}/remote-clusters-privileges.html#clusters-privileges-ccs-kibana[create a role for cross-cluster search privileges], and make sure the two roles have _identical_ names. Assign each role the following privileges:
+
+.. *Local cluster role*: Assign the `read` privilege to the indices you want to search, using _both_ the local and remote index patterns for each index. To specify a remote index, use the pattern `<remote_cluster_name>:<index_name>`.
++
+For example, if the remote cluster's name is `remote-security-data` and you want to query the `logs-*` indices, include both `logs-*` and `remote-security-data:logs-*` and assign them the `read` privilege.
++
+[role="screenshot"]
+image::images/ccs-local-role.png[Local cluster role configuration]
+
+.. *Remote cluster role*: Assign the `read` and `read_cross_cluster` privileges to the indices you want to search. You don't need to include the remote cluster's name here.
++
+[role="screenshot"]
+image::images/ccs-remote-role.png[Remote cluster role configuration]
+
+. On the local cluster:
+
+.. Assign the role you just created to a user who you want to configure your cross-cluster detection rules.
++
+[IMPORTANT]
+====
+* This step ensures that the privileges to read remote indices are applied from the user to the rule itself. When a user creates a new rule or saves edits to an existing rule, their current privileges are saved to the rule's API key. If that user’s privileges change in the future, you'll need to <<update-api-key,update the rule's API key>> to update the privileges that the rule uses.
+* This user must also have the <<enable-detections-ui,appropriate privileges>> to manage and preview rules.
+====
+
+.. With this user, <<rules-ui-create,configure a rule>> to search the remote indices. When creating or editing a rule, enter the `<remote_cluster_name>:<index_name>` pattern in the *Source* section.
```

Comment:
```suggestion
.. As this user, to <<rules-ui-create,configure a rule>> that searches remote indices: create or edit a rule, then enter the `<remote_cluster_name>:<index_name>` pattern in the *Source* section.
```

Reply:
I'd like to adopt much of your suggestion, but still keep the step as a more direct instruction ("do this" vs. "to do this, do that"). I think that tone is more consistent with most of our other steps. But your your other wording suggestions def make this a snappier step:
```suggestion
.. As this user, <<rules-ui-create,configure a rule>> that searches the remote indices: create or edit a rule, then enter the `<remote_cluster_name>:<index_name>` pattern in the *Source* section.
```